### DISCUSSION TITLE
why does xagent run through limit of 10000/min from the api key on gpt4.

### DISCUSSION THREAD

**Discussion by SmellyBones** [OpenBMB/XAgent#112] [Q&A] [answered]
based off of token limit and tpm rates, why does gpt-3.5 not do the job? on paper gpt3 should be the choice of running xagent i would like some help understanding why gpt 4 is needed and how i could posbily redcude my 10000/min limit that i keep running into very quickly. is this because i am asking to do a task that browses the internet and it stores to much data for its prompts?

**Comment by Ethan-Otto** [comment, marked as answer]
I'm unsure if this is your case, but the tier limit might be hit before your the model limits are reached. If you're in usage tier 1 you have a daily limit.  Though once you pay 50$ and wait 7 days this barrier is lifted. 

![image](https://github.com/OpenBMB/XAgent/assets/89815515/b6c3b757-d35b-4218-b260-9cf7d7f83f68)
